app/components/form/RequestBadge.tsx

```diff
@@ -3,11 +3,21 @@ import { $Enums } from '@prisma/client';
 import classNames from 'classnames';
 import CopyableButton from '@/components/generic/button/CopyableButton';
 
+interface DecisionData {
+  isTest?: boolean;
+}
+
 export default function RequestBadge({
   data,
   className,
 }: {
-  data?: { licencePlate: string; type: $Enums.RequestType; active: boolean; decisionStatus: $Enums.DecisionStatus };
+  data?: {
```

Comment:
let's re-organize the props so to have a single prop `request` which will have all information needed.